src/decorators/OrmConnection.ts

```diff
@@ -1,5 +1,6 @@
 import {ConnectionManager} from "typeorm";
 import {Container} from "typedi";
+import {Handler} from "typedi/types/Handler";
```

Comment:
this is "internal api". Everything should be imported from "typedi" namespace. We either need to export handler in typedi either not use it at all, this is possible in this case. See below comment.